### PR TITLE
Fix admin menu and submenu strings and packages

### PR DESCRIPTION
**Pull request by simun-ivanac**
Admin menu and submenu fixes regarding typos on packages

### CODE REVIEW

File: src/AdminMenus/AdminSubMenuCli.php [hunk 1]
```diff
@@ -109,7 +109,7 @@ public function __invoke(array $args, array $assocArgs) // phpcs:ignore
 			->renameTextDomain($assocArgs)
 			->searchReplaceString('example-parent-slug', $parentSlug)
 			->searchReplaceString('Admin Title', $title)
-			->searchReplaceString('Admin Menu Title', $menuTitle)
+			->searchReplaceString('Admin Sub Menu Title', $menuTitle)
```
Comment (goranalkovic-infinum) [inline src/AdminMenus/AdminSubMenuCli.php:112]:
```suggestion
			->searchReplaceString('Admin Submenu Title', $menuTitle)
```

File: src/AdminMenus/AdminSubMenuCli.php [hunk 2]
```diff
@@ -37,7 +37,7 @@ public function getDevelopArgs(array $args): array
 		return [
 			'parent_slug' => $args[1] ?? 'example-menu-slug',
 			'title' => $args[2] ?? 'Admin Title',
-			'menu_title' => $args[3] ?? 'Admin Title',
+			'menu_title' => $args[3] ?? 'Admin Sub Menu Title',
```
Comment (goranalkovic-infinum) [inline src/AdminMenus/AdminSubMenuCli.php:40]:
```suggestion
			'menu_title' => $args[3] ?? 'Admin Submenu Title',
```

**Review by goranalkovic-infinum — APPROVED** [review]
Couple of minor nitpicks

**Review by iruzevic — APPROVED** [review]
Great job 👍

File: src/AdminMenus/AdminSubMenuCli.php (hunk 1 above)
Comment (simun-ivanac) [reply src/AdminMenus/AdminSubMenuCli.php:112]: In the comments is used with space
Change everywhere or leave it?

**Review by dingo-d — APPROVED** [review]
Great catch!

**Pull request merged**